ide/vscode/launch-debug-probe.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Pico Debug Probe",
+            "cwd": "${workspaceRoot}",
+            "executable": "${command:cmake.launchTargetPath}",
+            "request": "launch",
+            "type": "cortex-debug",
+            "servertype": "openocd",
+            // This may need to be "arm-none-eabi-gdb" for some previous builds
+            "gdbPath" : "gdb-multiarch",
+            "serverArgs": [
+                "-c", "adapter speed 5000"
```

Comment:
This way https://github.com/vidadonizete/pivro/blob/main/environments/pico/.vscode/launch.json#L20C39-L20C39 it works too